lib/xendit_api/client.rb

```diff
@@ -102,6 +102,8 @@ def post(url, params, headers = {})
         req.body = params
       end
       response.body
+    rescue Faraday::ConnectionFailed
```

Comment:
I think we can rescue this kind of error exception at the application level. wdyt guys?

Reply:
I see, thank you for the insight